app/views/campaign_requests/_request_details.html.erb

```diff
@@ -24,11 +18,11 @@
     <div class="form-group">
       <span class="form-label control-label">
         <%= r.label :type_of_site do %>
-          Are you applying for the campaign platform or a bespoke microsite?<abbr title="required">*</abbr>
+          Are you applying for the single page campaign platform or a bespoke multi page site ?*<abbr title="required">*</abbr>
```

Comment:
I would remove the space between `site` and the `?` in `multi page site ?`... I know it was like that in the copy but...

Reply:
Done 😸

Reply:
thanks :)